src/test/scenarios/managed-network/output/src/managednetwork/azext_managednetwork/generated/_help.py

```diff
@@ -245,7 +245,7 @@
     examples:
       - name: Create/Update Managed Network Peering Policy
         text: |-
-               az managednetwork managed-network-peering-policy hub-and-spoke-topology create --managed-network-name "m\
+               az managednetwork managed-network-peering-policy hub-and-spoke-topology update --managed-network-name "m\
 yManagedNetwork" --policy-name "myHubAndSpoke" --type "HubAndSpokeTopology" --hub id="/subscriptions/subscriptionB/reso\

```

Comment:
You should also left out the discriminator. which is not necessarily be --type.
You can refer to https://github.com/Azure/autorest.az/blob/master/src/plugins/azgenerator/TemplateAzureCliCustom.ts#L174

Reply:
Also if it's scenario like repo-configuration in datafactory. you should also left out the discriminator key.